### PR TITLE
docs(projects): add e-learning

### PR DESCRIPTION
**Pull request by hasan-naim**
It's a basic website. I changed in two files one is `projects.json` and another one is `WEBSITE_BASIC.md`.

### CODE REVIEW

File: docs/WEBSITE_BASIC.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
   - [Currency Convertor](https://github.com/Roshaen/currency-converter) - [@Roshaen](https://github.com/Roshaen)
   - [Stickies](https://github.com/franciscoh017/stickies) - [@franciscoh017](https://github.com/franciscoh017)
   - [franciscoh017 Profile](https://github.com/franciscoh017/franciscoh017.github.io) - [@franciscoh017](https://github.com/franciscoh017)
-  - [Currency Converter](https://github.com/reliut-g/exchangerate-javascript-sdk) - [@reliut-g](https://github.com/reliut-g)
```
Comment (hasan-naim) [inline docs/WEBSITE_BASIC.md:5]: I didn't remove this line in my file but it is showing that I removed this line. I don't know what to do now.

Comment (ming-tsai) [reply docs/WEBSITE_BASIC.md:5]: Please remove this change, the bot will be place the project on the file

File: data/projects.json [hunk 2]
```diff
@@ -39,6 +39,16 @@
...
     },
+    {
+        "name": "eLearning",
+        "display": "eLearning",
+        "description": "It's a simple landing page. Made with Bootstrap 5.",
+        "author": "hasan-naim",
+        "category": "website",
+        "level": "basic",
+        "site": "https://elearning-hasan.netlify.app/",
+        "tags": "html, css, bootstrap"
```
Comment (ming-tsai) [inline data/projects.json:50]: Add `e-learning` tag for groundation of the project
```suggestion
        "tags": "e-learning, html, css, bootstrap"
```

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @hasan-naim,
Some suggestions for this pull request.

**Comment by hasan-naim** [comment]
Hey I have done what you have said. And I committed again. Can you check it?

**Pull request merged**